index.js

```diff
@@ -4,9 +4,7 @@
 var selectors = [
 	'iframe[src*="player.vimeo.com"]',
 	'iframe[src*="youtube.com"]',
-	'iframe[src*="youtube-nocookie.com"]',
-	'iframe[src*="kickstarter.com"][src*="video.html"]',
```

Comment:
I don't follow why the Kickstarter selector needs to be removed. I put together a [test case](http://jsbin.com/laxepudefa/edit?html,js,output) that only removes the `object` line, and it seems to work as expected. Is there something I'm missing?